sources/packages/api/src/services/education-program/education-program.service.ts

```diff
@@ -96,56 +100,75 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
+    }
     program.id = educationProgram.id;
     program.name = educationProgram.name;
     program.description = educationProgram.description;
-    program.credentialType = educationProgram.credentialType;
-    program.cipCode = educationProgram.cipCode;
-    program.nocCode = educationProgram.nocCode;
-    program.sabcCode = educationProgram.sabcCode;
-    program.regulatoryBody = educationProgram.regulatoryBody;
-    program.deliveredOnSite =
-      educationProgram.programDeliveryTypes.deliveredOnSite ?? false;
-    program.deliveredOnline =
-      educationProgram.programDeliveryTypes.deliveredOnline ?? false;
-    program.deliveredOnlineAlsoOnsite =
-      educationProgram.deliveredOnlineAlsoOnsite;
-    program.sameOnlineCreditsEarned = educationProgram.sameOnlineCreditsEarned;
-    program.earnAcademicCreditsOtherInstitution =
-      educationProgram.earnAcademicCreditsOtherInstitution;
-    program.courseLoadCalculation = educationProgram.courseLoadCalculation;
-    program.completionYears = educationProgram.completionYears;
-    program.hasMinimumAge = educationProgram.entranceRequirements.hasMinimumAge;
-    program.eslEligibility = educationProgram.eslEligibility;
-    program.hasJointInstitution = educationProgram.hasJointInstitution;
-    program.hasJointDesignatedInstitution =
-      educationProgram.hasJointDesignatedInstitution;
-    program.programStatus = educationProgram.programStatus;
-    program.institution = { id: educationProgram.institutionId } as Institution;
-    program.programIntensity = educationProgram.programIntensity;
-    program.institutionProgramCode = educationProgram.institutionProgramCode;
-    program.minHoursWeek = educationProgram.minHoursWeek;
-    program.isAviationProgram = educationProgram.isAviationProgram;
-    program.minHoursWeekAvi = educationProgram.minHoursWeekAvi;
-    program.minHighSchool = educationProgram.entranceRequirements.minHighSchool;
-    program.requirementsByInstitution =
-      educationProgram.entranceRequirements.requirementsByInstitution;
-    program.requirementsByBCITA =
-      educationProgram.entranceRequirements.requirementsByBCITA;
-    program.hasWILComponent = educationProgram.hasWILComponent;
-    program.isWILApproved = educationProgram.isWILApproved;
-    program.wilProgramEligibility = educationProgram.wilProgramEligibility;
-    program.hasTravel = educationProgram.hasTravel;
-    program.travelProgramEligibility =
-      educationProgram.travelProgramEligibility;
-    program.hasIntlExchange = educationProgram.hasIntlExchange;
-    program.intlExchangeProgramEligibility =
-      educationProgram.intlExchangeProgramEligibility;
-    program.programDeclaration = educationProgram.programDeclaration;
-    program.assessedBy = { id: educationProgram.userId } as User;
     if (!educationProgram.id) {
       program.submittedBy = { id: educationProgram.userId } as User;
     }
+    program.creator = { id: educationProgram.userId } as User;
```

Comment:
the creator should update only when the program is created not while editing, maybe you can move it inside 167